src/main/java/orca/flukes/xmlrpc/OrcaSMXMLRPCProxy.java

```diff
@@ -559,6 +560,58 @@ public String sliceStatus(String sliceId)  throws Exception {
 		return result;
 	}
 
+	@SuppressWarnings("unchecked")
+	public Boolean modifySliverSSH(String sliceId, String resId, String user, boolean sudo, List<String> keys) throws Exception {
+	    assert(sliceId != null);
```

Comment:
`assert()` shouldn't be used in public facing code.
[Objects.requreNonNull](https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html#requireNonNull(T)) from Java 7 might be the easiest thing to use.